<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Guide_Scene</class>
 <widget class="QWidget" name="Guide_Scene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guide_Scene</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/BIY_title_3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b1_newgame">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>301</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>是新游戏哦</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b3_editor">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>301</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>关卡编辑器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b2_setting">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>311</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>设定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b4_exit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>311</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
